mongomock/aggregate.py

```diff
@@ -626,6 +627,61 @@ def _handle_conditional_operator(self, operator, values):
             'aggregation pipeline, it is currently not implemented '
             ' in Mongomock.' % operator)
 
+    def _handle_control_flow_operator(self, operator, values):
+        if operator == '$switch':
+            if not isinstance(values, dict):
+                raise OperationFailure(
+                    '$switch requires an object as an argument, '
+                    'found: %s' % type(values)
+                )
+
+            branches = values.get('branches', [])
+            if not isinstance(branches, list):
+                raise OperationFailure(
+                    "$switch expected an array for 'branches', "
+                    'found: %s' % type(branches)
+                )
+            if len(branches) == 0:
```

Comment:
`if not branches` is equivalent and more pythonic

Reply:
Done, thanks!